hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/file/OMRecoverLeaseRequest.java

```diff
@@ -186,60 +179,56 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
     return omClientResponse;
   }
 
-  private String doWork(OzoneManager ozoneManager, long transactionLogIndex)
-      throws IOException {
-
+  private RecoverLeaseResponse doWork(OzoneManager ozoneManager,
```

Comment:
I think better to have restriction on some time gap between recoverLease and latest hsync call. Or else here recoverLease can be called immediately after hsync call.

Reply:
Since the patch is quite big now, the support of soft and hard limit is not implemented in this patch. We can file new JIRA to do that.

Reply:
Yes makes sense.